packages-next/keystone/src/lib/context/server-side-graphql-client.ts

```diff
@@ -119,7 +119,7 @@ const getItems = async ({
 }: {
   listKey: string;
   where?: Record<string, any> | null;
-  sortBy?: readonly string[] | null;
+  orderBy?: readonly Record<string, 'asc' | 'desc'>[] | null;
```

Comment:
This isn't right, the value can be anything, it will often be asc or desc but that is not guaranteed and there are definitely things that will deviate from it

Reply:
Ah, interesting, what's the correct type here then, is it `string`, or `any`?

Reply:
any

Reply:
> just FYI, I haven't made all the GraphQL API changes that we discussed so this will need more changing in the future

Yep, just want to get this close enough to run for now 👍